source/common/common/hash.h

```diff
@@ -39,6 +41,17 @@ class HashUtil {
     };
     return hash;
   }
+
+  /**
+   * Return 64-bit hash from murmur hash2 as is implemented in std::hash<string>.
+   * See
+   * (https://gcc.gnu.org/git/?p=gcc.git;a=blob_plain;f=libstdc%2b%2b-v3/libsupc%2b%2b/hash_bytes.cc).
+   * Which is based on (https://sites.google.com/site/murmurhash/).
+   * @param key supplies the string view
+   * @param seed the seed to use for the hash
+   * @return 64-bit hash representation of the supplied string view
+   */
+  static uint64_t murmurHash2_64(absl::string_view key, uint64_t seed = STD_HASH_SEED);
```

Comment:
Please add all of this code to some type of MurmurHash class, even if all the functions are static.